sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryStatusAndProgressSuite.scala

```diff
@@ -33,16 +33,10 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.streaming.StreamingQueryStatusAndProgressSuite._
 
 class StreamingQueryStatusAndProgressSuite extends StreamTest with Eventually {
-  implicit class EqualsIgnoreCRLF(source: String) {
```

Comment:
Replaces this with `assertJson` (see below) because this code made it harded to debug the differences between json strings. With `assertJson`, scala test would show nice diffs. 
```
[info] - StreamingQueryProgress - prettyJson *** FAILED *** (137 milliseconds)
[info]   "..."numRowsUpdated" : 1[,]
[info]       "memoryUsedByte..." did not equal "..."numRowsUpdated" : 1[]
[info]       "memoryUsedByte..." (StreamingQueryStatusAndProgressSuite.scala:213)
[info]   org.scalatest.exceptions.TestFailedException:
```